src/client/clipboard.cpp

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include <memory>
+#include "clipboard.h"
+
+#define XCLIP_SELECT "/usr/bin/xclip -selection c"
```

Comment:
Find a better way to do this. What if xclip is not in /usr/bin but in /usr/local/bin?